html-css/noel/style.css

```diff
@@ -0,0 +1,125 @@
+div {
+    background-color: #F7F9FB;
+    /* border-style: solid;
+    border-color: black; */
+    border-width: 2px;
+    margin: 5px;
+    padding: 5px; 
+}
+
+.main-content {
+    background-image: url("bgimage.png");
+}
+.container {
+    display: grid;
+    grid-template-rows: minmax(0, 1fr);
+    height: 96vh;
+    grid-template-columns: 2fr 8fr;
+}
+
+.main {
+    display: grid;
+    grid-template-rows: 1fr 8fr 1fr;
+    grid-gap: 1em;
+}
+
+body {
+    background-color: rgb(212, 208, 208);
+    font-family: Roboto, Helvetica, Arial;
+}
+
+nav ul {
+    display: grid;
+    grid-template-columns: 8fr 1fr 1fr 1fr 1fr;
+}
+
+/* .list {
```

Comment:
Please remove the unused css styles if no plans to be used later

Reply:
noted on this..thanks

Reply:
done on this..